code/ui/manager/src/components/sidebar/Tree.tsx

```diff
@@ -41,8 +41,8 @@ export const Action = styled.button(({ theme }) => ({
   display: 'inline-flex',
   alignItems: 'center',
   justifyContent: 'center',
-  width: 20,
-  height: 20,
+  width: 24,
+  height: 24,
```

Comment:
It looks like this width and height is also being applied to the expand/collapse buttons in the sidebar causing a layout shift. Can you change it to only apply to the status lights?

Reply:
This may also require realigning the status lights once the above change is made.